web/html/ParamsApp/Parameters.html

```diff
@@ -13,7 +13,7 @@
 </HEAD>
 
 <BODY>
-    <h1><img src="img/vionx-energy-logo-2.png" alt="Vionx logo" height="43" width="166">Cross-Module Parameter Manager</h1>
+    <h1><img src="img/largo-inc-logo-2.png" alt="Largo Inc logo" height="43" width="166">Parameter Manager</h1>
```

Comment:
Also, for future work, it occurs to me that we need to get the company specific stuff out of the open source project.